tap_xactly/streams.py

```diff
@@ -60,8 +60,6 @@ def sync(self):
                 self.client.setup_connection()
                 continue
 
-        self.client.close_connection()
```

Comment:
This was a really good catch finding this little bug! Moving this part out should make the whole stream operate more smoothly.